examples/usgs-3dep-13/n40w110/n41w106-13/n41w106-13.json

```diff
@@ -0,0 +1,118 @@
+        [
+          -104.999444445204,
+          41.0005555552955
+        ],
+        [
+          -106.000555555594,
+          41.0005555552955
+        ],
+        [
+          -106.000555555594,
+          39.9994444449061
+        ],
+        [
+          -104.999444445204,
+          39.9994444449061
+        ]
+      ]
+    ]
+  },
+  "links": [
+    {
+      "rel": "via",
+      "href": "https://prd-tnm.s3.amazonaws.com/StagedProducts/Elevation/13/TIFF/n41w106/USGS_13_n41w106.xml"
+    },
+    {
+      "rel": "root",
+      "href": "../../../catalog.json",
+      "type": "application/json"
+    },
+    {
+      "rel": "collection",
+      "href": "../../collection.json",
+      "type": "application/json"
+    },
+    {
+      "rel": "parent",
+      "href": "../catalog.json",
+      "type": "application/json"
+    }
+  ],
+  "assets": {
+    "data": {
+      "href": "https://prd-tnm.s3.amazonaws.com/StagedProducts/Elevation/13/TIFF/n41w106/USGS_13_n41w106.tif",
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+      "title": "USGS 1/3 arc-second n41w106 1 x 1 degree",
+      "description": "This tile of the 3D Elevation Program (3DEP) seamless products is 1/3 arc-second resolution.  3DEP data serve as the elevation layer of The National Map, and provide basic elevation information for Earth science studies and mapping applications in the United States. Scientists and resource managers use 3DEP data for global change research, hydrologic modeling, resource monitoring, mapping and visualization, and many other applications. 3DEP data compose an elevation dataset that consists of seamless layers and a high resolution layer.  Each of these layers consists of the best available raster elevation data of the conterminous United States, Alaska, Hawaii, territorial islands, Mexico and Canada.  3DEP data are updated continually as new data become available. Seamless 3DEP data are derived from diverse source data that are processed to a common coordinate system and unit of vertical measure. These data are distributed in geographic coordinates in units of decimal degrees, and in conformance with the North American Datum of 1983 (NAD 83). All elevation values are in meters and, over the conterminous United States, are referenced to the North American Vertical Datum of 1988 (NAVD 88). The vertical reference will vary in other areas. Seamless 3DEP data are available nationally (except for Alaska) at resolutions of 1 arc-second (approximately 30 meters) and 1/3 arc-second (approximately 10 meters). In most of Alaska, only lower resolution source data are available. As a result, most seamless 3DEP data for Alaska are at 2 arc-second (approximately 60 meters) grid spacing. Part of Alaska is available at the 1 and 1/3 arc-second resolutions from interferometric synthetic aperture radar (ifsar) collections starting in 2010.  Plans are in place for collection of statewide ifsar in Alaska. All 3DEP products are public domain.",
```

Comment:
This is a really long description. I think most of this belongs in the Collection description. Asset descriptions should be brief if needed at all.  title is usually sufficient.